.github/pull_request_template.md

```diff
@@ -23,4 +23,25 @@
 
 ### Cool Spaceship Picture
 
-<!-- optional but cool ->
+<!-- optional but cool -->
+
+<!-- Uncomment this section for status-go upgrade/dogfooding pull requests
```

Comment:
Looks good, but I would move it to above the `Cool Spaceship Picture` part

Reply:
done!